project-templates/node-json-vscode/README.md

```diff
@@ -0,0 +1,102 @@
+# Eclipse GLSP - Project Template: 🖥️ Node ● 🗂️ Custom JSON ● 🖼️ VS Code
```

Comment:
I think it would be nicer to have the section after the : in a second line:
```suggestion
# Eclipse GLSP - Project Template:<br> 🖥️ Node ● 🗂️ Custom JSON ● 🖼️ VS Code
```